### PR TITLE
Boards: add iMX8MM with SE050

### PR DESCRIPTION
**Pull request by mwasilew**
Signed-off-by: Milosz Wasilewski <milosz.wasilewski@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for e84703aa45739600e796515cf4c61ee0d0f07ea9 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/753/docs/artifacts/html/index.html

File: source/reference-manual/boards/se050_imx8mm.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+imx8mqevk as follows:
+
+Using four male to male jumper wires (Arduino Compatible Pin size)
+connect the two boards.
+
+.. figure:: /_static/boards/imx8mmevk_J1004.png
+     :width: 400
+     :align: center
+
+     imx8mqevk
```
Comment (Tim-Anderson) [inline source/reference-manual/boards/se050_imx8mm.rst:23]: this should be imx8mmevk

File: source/reference-manual/boards/se050_imx8mm.rst [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+|  Signal  | imx8mmevk  | OM-SE050ARD |
++==========+============+=============+
+| SCL      | J22 pin 4  | J2 pin 10   |
++----------+------------+-------------+
+| SDA      | J22 pin 1  | J2 pin 9    |
++----------+------------+-------------+
+| VDD_3V3  | J22 pin 2  | J8 pin 5    |
++----------+------------+-------------+
+| GND      | J22 pin 3  | J2 pin 7    |
++----------+------------+-------------+
```
Comment (Tim-Anderson) [inline source/reference-manual/boards/se050_imx8mm.rst:63]: The above table appears to be completely wrong. There is not documented J22 that I can find on the imxmevk and J22 on the se050 is a jumper block that does not appear to be wired to i2c.

File: source/reference-manual/boards/se050_imx8mm.rst [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+Attaching the SE050
+-------------------
+Connect the OM-SE050ARD Arduino Compatible Development Kit to the
+imx8mqevk as follows:
+
+Using four male to male jumper wires (Arduino Compatible Pin size)
+connect the two boards.
+
+.. figure:: /_static/boards/imx8mmevk_J1004.png
```
Comment (Tim-Anderson) [inline source/reference-manual/boards/se050_imx8mm.rst:19]: this picture is normally the one directly out of the QSG with the J1004 highlighted

File: source/reference-manual/boards/se050_imx8mm.rst [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+     :width: 400
+     :align: center
+
+     imx8mqevk
+
+.. figure:: /_static/boards/imx8mmevk_J1004_pinout.png
```
Comment (Tim-Anderson) [inline source/reference-manual/boards/se050_imx8mm.rst:25]: This picture should be zoomed in and pinouts shown like the used on the mq

File: source/reference-manual/boards/se050_imx8mm.rst [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+     :width: 400
+     :align: center
+
+     imx8mqevk i2c pinout
```
Comment (Tim-Anderson) [inline source/reference-manual/boards/se050_imx8mm.rst:29]: this should be 8mm not 8mq

**Review by Tim-Anderson — CHANGES_REQUESTED** [review]
I suggest you add Matt as a reviewer since he is the maintainer of the docs, you might also add David as a reviewer.

File: source/reference-manual/boards/se050_imx8mm.rst (hunk 2 above)
Comment (mwasilew) [reply source/reference-manual/boards/se050_imx8mm.rst:63]: ouch, yes. J22 is on SE050 (at lease silk screen says so, bottom side) and this is how @rsalveti appears to connect it.

**Comment by mwasilew** [comment]
I fixed all typos. Will update pictures on Monday

**Comment by doanac** [comment]
Docs for 15af2b7c92e07f14c0b4a55bb175f892acf3130f are browesable at: https://ci.foundries.io/projects/fio-docs/builds/754/docs/artifacts/html/index.html

File: source/reference-manual/boards/se050_imx8mm.rst (hunk 3 above)
Comment (mwasilew) [reply source/reference-manual/boards/se050_imx8mm.rst:19]: there are already some descriptions and arrows. I replaced it with a bit better quality picture

File: source/reference-manual/boards/se050_imx8mm.rst (hunk 4 above)
Comment (mwasilew) [reply source/reference-manual/boards/se050_imx8mm.rst:25]: Done

**Comment by mwasilew** [comment]
All comments addressed

**Comment by doanac** [comment]
Docs for a761e97640898cbe7b236d9e271c6de826c15e34 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/756/docs/artifacts/html/index.html

**Review by MatthewCroughan — APPROVED** [review]
Excellent. I'm really glad you've done this so fast. I do not have an SE050ARD to test this, but will soon. No reason to prevent merging though! :+1:

**Pull request merged**